scheduler/src/cook/rest/api.clj

```diff
@@ -2052,12 +2052,29 @@
   where \"...\" is a detailed error string describing the quota bounds exceeded."
   [conn {:keys [::job-pool-name-maps] :as ctx}]
   (let [db (db conn)
+        ; We cache quota by (user, pool) here because JobRouting plugins can result in
```

Comment:
This makes me sad as I'll be adding a cache at/behind the cook.quota level when doing the postgresql migration.